src/main/java/gift/repository/MemoryProductRepository.java

```diff
@@ -0,0 +1,49 @@
+package gift.repository;
+
+import gift.dto.ProductResponseDto;
+import gift.entity.Product;
+import org.springframework.stereotype.Repository;
+
+import java.util.*;
+
+@Repository
+public class MemoryProductRepository implements ProductRepository{
```

Comment:
가급적 IDE의 자동정렬 기능 ( mac 기준: `⌘ + ⌥ + L`, win 기준: `Ctrl + Alt + L`)을 적극적으로 활용해주시면 좋을 것 같아요. 

```suggestion
public class MemoryProductRepository implements ProductRepository {
```

흔히 영화나 드라마속에 나오는 개발을 떠올리며 개발은 혼자하는 것이라고 착각하기 쉽지만 
사실 개발은 함께하는 경우가 많아요. 😃
그렇기 때문에 내가 작성한 코드를 다른 팀원들이 잘 읽을 수 있도록하는 것도 코드를 유지보수하는데 많은 도움이될 수 있어요. 
상대적으로 사소해보일 수 있지만 이런 띄어쓰기 혹은 줄바꿈을 규칙적으로 하는 것도 이런 이유로 볼 수 있어요. 
이런 부분에 신경써서 코드를 작성해주면 팀원들이 코드를 더 잘 파악할 수 있게될거에요. ✨

이번 미션에서는 프로그래밍 요구사항에 따라서 미션을 진행해야하는데요. 
그 중 `google java coding convention`에 맞춰서  IDE에 지정해둔 뒤  [자동정렬되도록 셋팅](https://yeon-kr.tistory.com/197) 해봐도 좋을 것 같아요.

Reply:
보내주신 링크 따라서, 인텔리제이에 convention 파일 세팅했습니다. 자료 찾아 주셔서 감사합니다!!